src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiersTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks.processor;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.io.ByteArrayInputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.ZipInputStream;
+
+import static com.google.common.io.Resources.getResource;
+import static com.google.common.io.Resources.toByteArray;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ZipVerifiersTest {
+
+    private String publicKeyBase64 =
+        "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDEXfjyDFFigzsmFvTe2cWZ45ggH/XoS/3C6Ur/"
+            + "V0egi8k5hnIIgPEOUqhrX5UcQorSX7bIlMped6TtPkYdGs/QI6S5m2uz+6Mjai7ZfACGhYxIs8"
+            + "35msqvRsDM0tIle/h3eZJb7iPE0anMWb8MkBYU3D3vAnPdBZxiEIwNMUNzqQIDAQAB";
+    
+    @Test
+    public void should_verify_signed_file_successfully() throws Exception {
+        byte[] test1PdfBytes = toByteArray(getResource("test1.pdf"));
+        byte[] test1SigPdfBytes = toByteArray(getResource("signature/test1.pdf.sig"));
```

Comment:
This is incorrect. Signature is generated for zip file and not individual pdf's.

Reply:
That is just a signature test to verify the algorithm. And it is correct in its context (unit test).

Reply:
well this gives the impression that you are generating signatures for pdf which is incorrect

Reply:
That is not what I am testing. Sorry if it gives you the wrong impression. The test is useful as it is.